### PR TITLE
🌐 [i18n-KO] docs: ko: Translate `multiple_choice.mdx`

### PR DESCRIPTION
**Pull request by gabrielwithappy**
# What does this PR do?

Translated the `multiple_choice.mdx` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)
Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)
@sgugger, @ArthurZucker, @eunseojo May you please review this PR?

### CODE REVIEW

**Comment by gabrielwithappy** [comment]
Team PseudoLab, may you please review this PR? @0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 1]
```diff
@@ -0,0 +1,461 @@
...
+ 'sent2': 'A drum line',
+ 'startphrase': 'Members of the procession walk down the street holding small horn brass instruments. A drum line',
+ 'video-id': 'anetv_jkn6uvmqwh4'}
+```
+
+여기에는 많은 필드가 있는 것처럼 보이지만 실제로는 매우 간단합니다:
+
+- `sent1` 및 `sent2`: 이 필드는 문장이 어떻게 시작되는지 보여주며, 이 두 필드를 합치면 `startphrase` 필드가 됩니다.
+- `어미(ending)`: 문장이 어떻게 끝날 수 있는지에 대한 가능한 어미를 제시하지만 그 중 하나만 정답입니다.
+- `레이블`: 올바른 문장 어미를 식별합니다.
```
Comment (HanNayeoniee) [inline docs/source/ko/tasks/multiple_choice.mdx:80]: - `레이블(label)`: 올바른 문장 어미를 식별합니다.

영어를 병기해서 label이 데이터의 구성에 해당하는 것을 알려주는건 어떨까요?

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+
+## 전처리[[preprocess]]
+
+다음 단계는 문장의 시작과 네 가지 가능한 어미를 처리하기 위해 BERT 토큰화기를 로드하는 것입니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/tasks/multiple_choice.mdx:84]: 다음 단계로 문장의 시작과 네 가지 가능한 어미를 처리하기 위해 BERT 토크나이저를 로드합니다:

- 토큰화기 -> 토크나이저
- 번역 느낌이 나는 "~하는 것입니다" 문장을 자연스러운 한국어로 수정해봤습니다.

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+...     ]
+
+...     first_sentences = sum(first_sentences, [])
+...     second_sentences = sum(second_sentences, [])
+
+...     tokenized_examples = tokenizer(first_sentences, second_sentences, truncation=True)
+...     return {k: [v[i : i + 4] for i in range(0, len(v), 4)] for k, v in tokenized_examples.items()}
+```
+
+전체 데이터 집합에 전처리 기능을 적용하려면 🤗 Datasets [`~datasets.Dataset.map`] 메서드를 사용합니다. `batched=True`를 설정하여 데이터 집합의 여러 요소를 한 번에 처리하면 `map` 함수의 속도를 높일 수 있습니다:
```
Comment (HanNayeoniee) [inline docs/source/ko/tasks/multiple_choice.mdx:116]: 전체 데이터 집합에 전처리 기능을 적용하려면 🤗 Datasets [`~datasets.Dataset.map`] 메소드를 사용합니다. `batched=True`를 설정하여 데이터 집합의 여러 요소를 한 번에 처리하면 `map` 함수의 속도를 높일 수 있습니다:

`method`를 `메소드`로 번역해야 저희 glossary와 통일성이 있어 보입니다!

**Review by HanNayeoniee — COMMENTED** [review]
수고하셨습니다! 즐거운 연휴 보내세요 🙂

File: docs/source/ko/tasks/multiple_choice.mdx (hunk 3 above)
Comment (gabrielwithappy) [reply docs/source/ko/tasks/multiple_choice.mdx:116]: checked :-)

File: docs/source/ko/tasks/multiple_choice.mdx (hunk 2 above)
Comment (gabrielwithappy) [reply docs/source/ko/tasks/multiple_choice.mdx:84]: checked.
불러옵니다 로 추가 수정했습니다.
감사합니다. :-)

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 4]
```diff
@@ -0,0 +1,461 @@
...
+... )
+```
+
+[`compile`](https://keras.io/api/models/model_training_apis/#compile-method)을 사용하여 훈련 모델을 구성합니다:
+
+```py
+>>> model.compile(optimizer=optimizer)
+```
+
+훈련을 시작하기 전에 설정해야 할 마지막 두 가지는 예측의 정확도를 계산하고 모델을 Hub로 푸시하는 방법을 제공하는 것입니다. 이 두 가지 작업은 모두 [Keras 콜백](../main_classes/keras_callbacks)을 사용하여 수행할 수 있습니다.
```
Comment (0525hhgus) [inline docs/source/ko/tasks/multiple_choice.mdx:344]:
```suggestion
훈련을 시작하기 전에 설정해야 할 마지막 두 가지는 예측의 정확도를 계산하고 모델을 허브로 푸시하는 방법을 제공하는 것입니다. 이 두 가지 작업은 모두 [Keras 콜백](../main_classes/keras_callbacks)을 사용하여 수행할 수 있습니다.
```
해당 문서에서 `허브`로 많이 사용되어 `허브`로 변경했습니다!

**Review by 0525hhgus — CHANGES_REQUESTED** [review]
좋은 번역 감사합니다! 태스크의 특징이 잘 드러난 번역이었다고 생각합니다 👍 
아래 리뷰를 남겼습니다 😄

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 5]
```diff
@@ -0,0 +1,461 @@
...
+
+```py
+>>> from huggingface_hub import notebook_login
+
+>>> notebook_login()
+```
+
+## SWAG 데이터 세트 불러오기[[load-swag-dataset]]
+
+먼저 🤗 Datasets  라이브러리에서 SWAG 데이터셋의 '일반' 구성을 불러옵니다:
```
[the remark this answers is not in the document]
Comment (sim-so) [reply docs/source/ko/tasks/multiple_choice.mdx:51]:
```suggestion
먼저 🤗 Datasets  라이브러리에서 SWAG 데이터 세트의 '일반' 구성을 가져옵니다:
```

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 6]
```diff
@@ -0,0 +1,461 @@
...
+ 'sent2': 'A drum line',
+ 'startphrase': 'Members of the procession walk down the street holding small horn brass instruments. A drum line',
+ 'video-id': 'anetv_jkn6uvmqwh4'}
+```
+
+여기에는 많은 필드가 있는 것처럼 보이지만 실제로는 매우 간단합니다:
+
+- `sent1` 및 `sent2`: 이 필드는 문장이 어떻게 시작되는지 보여주며, 이 두 필드를 합치면 `startphrase` 필드가 됩니다.
+- `어미(ending)`: 문장이 어떻게 끝날 수 있는지에 대한 가능한 어미를 제시하지만 그 중 하나만 정답입니다.
+- `레이블(label)`: 올바른 문장 어미를 식별합니다.
```
Comment (sim-so) [inline docs/source/ko/tasks/multiple_choice.mdx:80]: `ending`이 `어미`라는 뜻도 있지만 여기에서는 `종결(어구/어절)`에 더 가깝게 느껴집니다.
`종결 구절`은 너무 딱딱한 표현이라 고민되는데, `어미`는 문법적 의미이니 다른 표현을 고민해보면 좋을 것 같아요.
그리고 `ending`과 `label`의 한국어 표현을 써주신 것처럼, `startphrase`도 한국어 표현을 추가하는 것은 어떨까요?

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 7]
```diff
@@ -0,0 +1,461 @@
...
+...         batch = {k: tf.reshape(v, (batch_size, num_choices, -1)) for k, v in batch.items()}
+...         batch["labels"] = tf.convert_to_tensor(labels, dtype=tf.int64)
+...         return batch
+```
+</tf>
+</frameworkcontent>
+
+## 평가 하기[[evaluate]]
+
+훈련 중에 메트릭을 포함하면 모델의 성능을 평가하는 데 도움이 되는 경우가 많습니다. 🤗[Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하여 평가 방법을 빠르게 불러올 수 있습니다. 이 작업에서는 [accuracy](https://huggingface.co/spaces/evaluate-metric/accuracy) 지표를 로드합니다(🤗 Evaluate [빠른 둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하여 지표를 불러오고 계산하는 방법에 대해 자세히 알아보세요):
```
Comment (sim-so) [inline docs/source/ko/tasks/multiple_choice.mdx:215]:
```suggestion
훈련 중에 메트릭을 포함하면 모델의 성능을 평가하는 데 도움이 되는 경우가 많습니다. 🤗[Evaluate](https://huggingface.co/docs/evaluate/index) 라이브러리를 사용하여 평가 방법을 빠르게 가져올 수 있습니다. 이 작업에서는 [accuracy](https://huggingface.co/spaces/evaluate-metric/accuracy) 지표를 가져옵니다(🤗 Evaluate [둘러보기](https://huggingface.co/docs/evaluate/a_quick_tour)를 참조하여 지표를 가져오고 계산하는 방법에 대해 자세히 알아보세요):
```
1. 현재 glossary 기준으로 수정합니다.
2. transformers 기준으로 `quick tour`는 `둘러보기`로 되어 있어서 맞추었습니다.

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 8]
```diff
@@ -0,0 +1,461 @@
...
+
+<frameworkcontent>
+<pt>
+<Tip>
+
+[`Trainer`]로 모델을 미세 조정하는 데 익숙하지 않다면 기본 튜토리얼 [여기](../training#train-with-pytorch-trainer)를 살펴보세요!
+
+</Tip>
+
+이제 모델 훈련을 시작할 준비가 되었습니다! [`AutoModelForMultipleChoice`]로 BERT를 로드합니다:
```
Comment (sim-so) [inline docs/source/ko/tasks/multiple_choice.mdx:247]:
```suggestion
이제 모델 훈련을 시작할 준비가 되었습니다! [`AutoModelForMultipleChoice`]로 BERT를 가져옵니다:
```

File: docs/source/ko/tasks/multiple_choice.mdx [hunk 9]
```diff
@@ -0,0 +1,461 @@
...
+훈련이 완료되면 모든 사람이 모델을 사용할 수 있도록 [`~transformers.Trainer.push_to_hub`] 메소드를 사용하여 모델을 허브에 공유하세요:
+
+```py
+>>> trainer.push_to_hub()
+```
+</pt>
+<tf>
+<Tip>
+
+Keras로 모델을 미세 조정하는 데 익숙하지 않다면 기본 튜토리얼 [여기](../training#train-a-tensorflow-model-with-keras)을 살펴보시기 바랍니다!
```
Comment (sim-so) [inline docs/source/ko/tasks/multiple_choice.mdx:297]:
```suggestion
Keras로 모델을 미세 조정하는 데 익숙하지 않다면 기본 튜토리얼 [여기](../training#train-a-tensorflow-model-with-keras)를 살펴보시기 바랍니다!
```
`여기` 뒤에 오는 조사를 `를`로 바꾸었습니다.

**Review by sim-so — CHANGES_REQUESTED** [review]
새로운 내용이 많은 문서였어요!
즐거운 연휴를 보내셨길 바라며, 몇 가지 수정 제안을 보내드립니다 😸

File: docs/source/ko/tasks/multiple_choice.mdx (hunk 6 above)
Comment (KIHOON71) [reply docs/source/ko/tasks/multiple_choice.mdx:80]: 후행어절 혹은 후행문장 같은 표현은 어떨까요?

Comment (wonhyeongseo) [reply docs/source/ko/tasks/multiple_choice.mdx:80]:
```suggestion
- `sent1` 및 `sent2`: 두 필드는 문장이 어떻게 시작되는지 나타내며, 합치면 `첫 구절(startphrase)` 필드가 됩니다.
- `마지막 구절(ending)`: 문장이 끝날 수 있는 수많은 구절 중 하나만 정답으로 삼습니다.
- `레이블(label)`: 문장에 알맞은 마지막 구절을 식별합니다.
```

Comment (gabrielwithappy) [reply docs/source/ko/tasks/multiple_choice.mdx:80]: 구절은 온전하기 끝나지 않는 문장으로 구절이 문서에 가장 적합한 것 같습니다.
리뷰 감사드립니다. :-)

**Comment by gabrielwithappy** [comment]
@sgugger, @ArthurZucker, @eunseojo 
May you please review this PR?

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!
